core-plugins/docs/Validator-transform.md

```diff
@@ -138,34 +142,36 @@ field is greater than 1MB.
 It has been "pretty-printed" for readability. It uses the CoreValidator and references functions 
 using their JavaScript names (such as ``coreValidator.isDate``):
 
-    {
-        "name": "Validator",
-        "type": "transform",
-        "properties": {
-            "validators": "core",
-            "validationScript": "function isValid(input, context) {
-                                  var isValid = true;
-                                  var errMsg = \"\";
-                                  var errCode = 0;
-                                  var coreValidator = context.getValidator(\"coreValidator\");
-                                  var metrics = context.getMetrics();
-                                  var logger = context.getLogger();
-                                  if (!coreValidator.isDate(input.date)) {
-                                     isValid = false; errMsg = input.date + \"is invalid date\"; errCode = 5;
-                                     metrics.count(\"invalid.date\", 1);
-                                  } else if (!coreValidator.isUrl(input.url)) {
-                                     isValid = false; errMsg = \"invalid url\"; errCode = 7;
-                                     metrics.count(\"invalid.url\", 1);
-                                  } else if (!coreValidator.isInRange(input.content_length, 0, 1024 * 1024)) {
-                                     isValid = false; errMsg = \"content length >1MB\"; errCode = 10;
-                                     metrics.count(\"invalid.body.size\", 1);
-                                  }
-                                  if (!isValid) {
-                                    logger.warn(\"Validation failed for record {}\", input);
-                                  }
-                                  return {'isValid': isValid, 'errorCode': errCode, 'errorMsg': errMsg};
-                                };"
-        }
+```javascript
```

Comment:
should this be js or javascript? or are they interchangeable?

Reply:
It is [interchangeable](https://prismjs.com/#supported-languages). We can either user `js` or `javascript`